experimental/class-brand-check-expression.md

```diff
@@ -0,0 +1,21 @@
+# [class.hasInstance](https://github.com/tc39/proposal-class-brand-check)
+
+## ClassExpression
+
+```js
+extend interface classHasInstanceExpression <: Expression {
+    type: "classHasInstanceExpression",
+    source: Expression
```

Comment:
```suggestion
    instance: Expression
```

The `ImportExpression` has `source: Expression` because `"some-library"` is the imported source in `import("some-library")`. But I find `source` in `classHasInstanceExpression` confusing.
Other suggestions are welcome.

Reply:
`candidate` is my suggestion. `argument` is also possible, but the syntax suggests that additional arguments may be added, so `argument` would become ambiguous.

Reply:
Maybe `instance` is more appropriate, I hope to express the 
following infomation: 
1. `class.hasInstance` is not a real funtion object, not support multiple arguments. 
2.  The user can understand the role of `class.hasInstance` like `instanceof`. I hope this way could reduce the cost of understanding.